contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -568,13 +593,18 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     function _sendTeleporterMessage(
         TeleporterMessageInput memory messageInput,
         TeleporterMessageReceipt[] memory receipts
-    ) private returns (uint256) {
+    ) private returns (bytes32) {
+        // If the blockchain ID has yet to be initialized, do so now.
+        bytes32 blockchainID_ = _initializeBlockchainID();
+
         // Get the message ID to use for this message.
-        uint256 messageID = _getNextMessageID(messageInput.destinationBlockchainID);
+        uint256 messageNonce_ = messageNonce;
+        bytes32 messageID_ =
```

Comment:
my understanding is we add `_` to suffix when it's repeated with a state variable, but we don't have a state variable `messageID` so might not be needed, but no strong pref here

Reply:
Removed, agree to keep that convention consistent as only where required.